### ISSUE TITLE
Add support for contigs as well as reads

### ISSUE THREAD

**Issue by tseemann** [ODiogoSilva/assemblerflow#42]
It would be useful to allow input of mixed read sets and contigs.
Not all genomes are available as reads.

**Comment by ODiogoSilva** [comment]
Depending on the starting process, the input can be either fastq, fasta or fasta+fastq. For instance, a pipeline that starts with `chewbbaca` for allele call and `abricate` for ARM will require fasta as inputs. A pipeline that starts with an assembly module will require fastq input. A pipeline that starts with an assembly polisher like `pilon` will require the assembly fastas and the corresponding fastq reads. These are provided to the pipeline via the `--fastq` and `--fasta` parameters. Or did I misunderstand the issue?

**Comment by tseemann** [comment]
Sorry, I must have misunderstood @kristyhoran when we were talking about. I thought it only accepted raw FASTQ data.

**Comment by tseemann** [comment]
If i have 10 SRA samples (FASTQ reads), and 5 Genbank (FASTA contigs) - can one create a single workflow that wil produce a cgMLST result for all 15 samples?

**Comment by ODiogoSilva** [comment]
Yes, that case is still not possible, but it's a good feature and something that should be easy to implement in the current build. The input channels for chewbbaca, and other `fasta` receiving processes, can be set to join the `fasta` data from the preceding process with the data provided via the `--fasta` option. It should not take long to implement it.

There is no requirement for `integrity_coverage` to be the first process for `fastqc` input (other than the spades process requiring it). A pipeline can start right away with skesa, trimmomatic, etc.

**Comment by ODiogoSilva** [comment]
This feature has been implemented in a branch and is described in https://github.com/ODiogoSilva/assemblerflow/pull/51. We're now performing several tests with it. Could you check it to see if it meets your needs?

**Comment by ODiogoSilva** [comment]
Another useful scenario for this feature, is the ability to run an assembly pipeline from SSR accessions, fastq and fasta. In this case, the pipeline can start with the `reads_download` process (which basically downloads fastq reads) and then we provide extra_inputs for the beginning of the assembly pipeline and other processes of interest:

```
reads_download integrity_coverage={'extra_input':'fastq'} (...) chewbbaca={'extra_input':'fasta'}
```

And execute the pipeline like:

```
nextflow run (...) --accessions path/to/accessions --fastq path/to/fastq --chewieInput path/to/fasta
```

This would allow allele call to be performed from acessions, fastq reads and fasta in a single workflow.

**Comment by ODiogoSilva** [comment]
Closed via #51
